workers/unity/Packages/io.improbable.gdk.core/Worker/WorkerConnector.cs

```diff
@@ -145,6 +147,7 @@ protected async Task Connect(IConnectionHandlerBuilder builder, ILogDispatcher l
 
         protected virtual void HandleWorkerConnectionEstablished()
         {
+            protocolLog?.OnWorkerConnected(Worker.World);
```

Comment:
Is the intention that the user can optionally call `base.HandleWorkerConnectionEstablished()` to wire this up?

Reply:
I assume users will call `base.HandleWorkerConnectionEstablished()` when overriding the method. Should I use the `OnWorkerCreationFinished` event instead?

Reply:
You could even call this directly before we call `HandleWorkerConnectionEstablished` in `Connect` rather than the indirection through the callbacks